samples/net/google_iot_device/src/main.c

```diff
@@ -26,6 +28,7 @@ void main(void) {
     DeviceID>/state
     */
 
+#if 0
```

Comment:
Please remove lines in `#if 0 - #endif` blocks.

Reply:
> Please remove lines in #if 0 - #endif blocks.

Done.